ci/ray_ci/automation/weekly_green_metric.py

```diff
@@ -48,7 +49,7 @@ def main(production: bool, check: bool) -> None:
         logger.info("Weekly green metric updated successfully")
 
     if check and blockers.totalCount != 0:
-        raise Exception(f"Found {blockers.totalCount} release blockers")
+        sys.exit(42)  # Not retrying the check on Buildkite jobs
```

Comment:
could you print a log line to stderr before exiting? the original message is actually good, just don't throw the exception (so that it will not be printing the stacktrace).

Reply:
done. I can also use `logger.error(...)` if you prefer it that way

Reply:
Either is fine.

But could you remove the "Please fix before proceeding."? I am confused how one can "proceed" after "fix" in the context here.

Reply:
fixed!